src/Servers/HttpSys/src/HttpSysListener.cs

```diff
@@ -238,14 +238,14 @@ private void Dispose(bool disposing)
                     {
                         return;
                     }
-                    Logger.LogTrace("Disposing the listener.");
+                    Logger.LogTrace(LoggerEventIds.ListenerDisposing, "Disposing the listener.");
 
                     Stop();
                     DisposeInternal();
                 }
                 catch (Exception exception)
                 {
-                    Logger.LogError(0, exception, "Dispose");
+                    Logger.LogError(LoggerEventIds.ErrorInDispose, exception, "Dispose");
```

Comment:
Also consistent ordering?
```suggestion
                    Logger.LogError(LoggerEventIds.ListenerDisposeError, exception, "Dispose");
```

Reply:
made all events to look as Error.